### DISCUSSION TITLE
OCR on images in issues

### DISCUSSION THREAD

**Discussion by adriens** [community/community#10932] [Projects and Issues]
# 👉 Context

More and more, everyday, we are using GH Issues in replacement of Jira with a great success. People put more and more content which is a great thing, and to preserve some contents look & feel (email essentially in our case, or third party issue managing systems).

# 🗣️ Consequence

 -  This approach is very convenient, but the drawback is that the **text content embedded is the screenshot is not searchable**.

# 💡 Idea

- The idea would to be make GH Issue to parse the attached image and put the grabbed text in a section next to the screenshot, hence making it searchable in the issue itself.

# 🙏 Thanks

That would be awesome and would make us even more efficient with GH Issues.

**Comment by adriens** [comment]
BTW, it works natively on MacOS/Safari and it provides an amazin experience : https://www.macworld.com/article/351669/macos-monterey-how-to-live-text-look-up-photos-quick-look-safari.html

**Comment by Dougniel** [comment]
That's what it looks like with Safari on macOS Monterey 🎥  :
https://user-images.githubusercontent.com/39691264/157805127-cc210511-086f-4eeb-9f73-efef288a2bda.mp4

There's also a pure JS lib that seems to do that : https://tesseract.projectnaptha.com 😃
